src/main/resources/swagger/api-docs.yaml

```diff
@@ -6632,8 +6632,8 @@ paths:
       tags:
         - Workspaces
       summary: |
-        Get all tags used in FireCloud (for autocomplete)
-      description: list of all tags used in FireCloud and their associated frequencies
+        Get all workspace tags (for autocomplete)
+      description: Get all tags for workspaces that a user has access to (for autocomplete)
```

Comment:
Since I'm here, I harmonized this text with https://github.com/broadinstitute/rawls/pull/1689